src/Pickles/Pickles.TestFrameworks/MsTest/MsTestSingleResults.cs

```diff
@@ -137,6 +137,10 @@ public override TestResult GetExampleResult(ScenarioOutline scenario, string[] e
             var scenarioElements = this.GetScenariosForScenarioOutline(scenario);
 
             var theScenario = this.GetScenarioThatMatchesTheExampleValues(scenario, exampleValues, scenarioElements);
+            if (theScenario == null)
```

Comment:
Thanks for this fix. I'm following a test-first policy in Pickles, so please provide some tests for this case. It is worth checking out the other test result providers because the same problem may be lurking there.

See [CONTRIBUTING.md](https://github.com/picklesdoc/pickles/blob/develop/CONTRIBUTING.md) for information on how to contribute code to the test result providers.